app/scripts/components/searchdetails.js

```diff
@@ -128,6 +117,58 @@ var exports = (function () {
           });
           return parameters;
         }
+      },
+      createdAfterValid: {
+        get: function () {
+          return this.createdAfter === "" || moment(this.createdAfter, "YYYY-MM-DD", true).isValid();
+        }
+      },
+      createdBeforeValid: {
+        get: function () {
+          return this.createdBefore === "" || moment(this.createdBefore, "YYYY-MM-DD", true).isValid();
+        }
+      }
+    },
+
+    watch: {
```

Comment:
This actually allows for a much lower polling of the server: we now do a search every time our params update - the store is now ready to do a lot less polling.